package.json

```diff
@@ -38,6 +37,7 @@
   },
   "homepage": "https://github.com/craftship/yith#readme",
   "dependencies": {
+    "aws-sdk": "^2.7.10",
```

Comment:
Just a heads up, the reason this is not in `dependencies` is that the Lambda execution environment already includes this by default.  Only required locally when serving the functions via the Serverless webpack plugin.

Reference: http://docs.aws.amazon.com/lambda/latest/dg/current-supported-versions.html